zerver/migrations/0018_auto_20160611_1648.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.core.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0017_userprofile_bot_type'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='subscription',
+            name='starred',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AlterField(
```

Comment:
I believe @timabbott fixed this today. Could you please rebase and re-generate the migration?